README.md

```diff
@@ -1 +1,68 @@
-# financialMetrics
\ No newline at end of file
+# financial-metrics
+
+- The app takes information from the [Financial Modeling Prep API](https://site.financialmodelingprep.com/developer/docs).
+- To get basic information I first ran the end point Financial Statement Symbol.
+- To filter companies I based from the forber article [The Global 2000](https://www.forbes.com/lists/global2000/?sh=4cd60b605ac0)
+- The API doesn't give access to some of the companies on that list. I end up chossing 30 to render the front screen.
+- A local array was created with the basic information:
+  > Company name.
+
+  > Symbol.
+
+  > Image.
+- The images were taken from the companies sites
+- To access the company details I used the end point Income Statement. It works with the company symbol.
+- Based on Nelson [Sakwa on Behance](https://www.behance.net/sakwadesignstudio) design.
+
+## Built With:
+
+- Javascrip
+- React Redux
+- Github and Gitflow
+
+## Video Presentation:
+
+- [click here](https://www.loom.com/share/77e6794441274926a0fb91ddd286d0a9)
+
+## Live Demo
+
+[Financial Metrics](https://peppy-tarsier-ec49cb.netlify.app/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps:
+
+- It's necessary to install the LTS option of Node.js. [link](https://nodejs.org/en/).
+- npm will be included in the previous installation.
+- Go to the folder where you want to store the project, and clone the repository:
+- SSH: https://github.com/zkr024/financial-metrics.git
+- From the folder where all the files are located please Run npm install.
+- And to launch the web server, please run npm start.
+
+## Author
+
+👤 Oscar Bermudez
+
+- GitHub: [@githubhandle](https://github.com/zkr024)
+- Twitter: [@twitterhandle](https://twitter.com/zkr024)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/oscar-bermudez-07908222a)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] **Required:** You did well by mentioning the owner of the original design in your presentation and the PR summary. However, you have not included it in your documentation's `Acknowledgement` section. Kindly add the original designer's name here. This ensures that credit is given to the due person. It is also a professional norm. I hope this makes sense. 😏

Reply:
Thank you for the feedback. I updated the readme file. But I'm getting the submit message instead of the required changes in the dashboard... that means that is all set?
![image](https://user-images.githubusercontent.com/93548973/169917254-a5abd429-8e3b-42ad-9152-16ccb363a986.png)